.travis.yml

```diff
@@ -79,7 +79,7 @@ install:
     - bash miniconda.sh -b -p $HOME/miniconda
     - export CONDA_HOME="$HOME/miniconda"
     - export PATH="$CONDA_HOME/bin:$PATH"
-    - pip install cwrap
+    - pip install -r requirements.txt
```

Comment:
The family `libecl - libres - ert` are built together on travis, based on this https://github.com/Statoil/ert/blob/master/travis/build_total.py

Reply:
Så meine du at det er feil å installere det som er i requirements.txt? Når ein legger til bruk av nye bibliotek så må travis få beskjed på eit vis.

Reply:
Feil og feil; men poenget er at for en vilkårlig PR i ett av repoene libecl, libres og ert så bygges alltid *hele kjeden*, basert på ett lite Python script som ligger i ert repoet. Det er derfor det scriptet som må oppdateres: https://github.com/Statoil/ert/blob/master/travis/install_python_packages.py

Så også for en libecl PR må vi få travis til å innstallere `jinja2` - og da hjelper det ikke i legge inn `pip install -r requirements.txt` slik du har gjort.

Reply:
(I guess I should have kept this in english...) Sorry, but I didn't  understand what you where aiming at. I can make the changes in the install_python_packages script.

Reply:
> I can make the changes in the install_python_packages script.

That is exactly what I am aiming at.

Reply:
Sorry in advance for polluting the review of this PR. I find it a bit unfortunate that `build_total.py` uses a dedicated `install_python_packages.py` instead of pip installing the `requirements.txt` from the 3 repos. Was there a particular reason for doing that?

Reply:
Don't know - at some time `pip install` was not very efficient/friendly with binary packages like `numpy`. Maybe it could be replaced with three `pip install -r $repo/requirements.txt` calls now.

Reply:
There are still some things not available in pip, like pyqt4